variants/Aventen_S3_Sync/pins_arduino.h

```diff
@@ -0,0 +1,71 @@
+#ifndef Pins_Arduino_h
+#define Pins_Arduino_h
+
+#include <stdint.h>
+#include "soc/soc_caps.h"
+
+#define USB_VID 0x303a
+#define USB_PID 0x1001
+#define USB_MANUFACTURER "Aventen"
+#define USB_PRODUCT "Aventen S3 Sync"
+#define USB_SERIAL ""
+
+#define EXTERNAL_NUM_INTERRUPTS 46
+#define NUM_DIGITAL_PINS        48
+#define NUM_ANALOG_INPUTS       20
+
+#define analogInputToDigitalPin(p)  (((p)<20)?(analogChannelToDigitalPin(p)):-1)
+#define digitalPinToInterrupt(p)    (((p)<48)?(p):-1)
+#define digitalPinHasPWM(p)         (p < 46)
```

Comment:
This will create problems when using pin 48 for interrupts. Take a look at #8562 for the recommended changes.